specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -178,7 +175,7 @@ export function Notifications(): JSX.Element {
            */}
           <p>{commonText('mostRecentNotificationsTop')}</p>
           {notifications.map((notification, index) => (
-            <ErrorBoundary dismissable>
+            <ErrorBoundary dismissable key={index}>
               <NotificationComponent
                 key={index}
```

Comment:
This `key` attribute is no longer needed as it is no longer the top-level component in the loop. Remove the line